test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1272,6 +1272,27 @@ UCP_INSTANTIATE_TEST_CASE_TLS(select_ep_transport, ib, "ib")
 UCP_INSTANTIATE_TEST_CASE_TLS(select_ep_transport, ib_shm, "ib,shm")
 UCP_INSTANTIATE_TEST_CASE_TLS(select_ep_transport, all, "all")
 
+
+class select_transport_rndv : public test_ucp_wireup {
+public:
+    static void get_test_variants(std::vector<ucp_test_variant> &variants)
+    {
+        add_variant_with_value(variants, UCP_FEATURE_TAG, TEST_TAG, "tag");
+    }
+};
+
+UCS_TEST_P(select_transport_rndv, select_dc, "NUM_EPS=300")
```

Comment:
1. add comment to explain the test idea
2. does this test fail if we remove the fixes from select.c?

Reply:
added UCX_RC_MAX_NUM_EPS=1000 to fail the test when the fix is out